### DISCUSSION TITLE
想要在收到弹幕时加一段处理逻辑，应该改哪里？

### DISCUSSION THREAD

**Discussion by nudoo** [xfgryujk/blivechat#193] [提问] [answered]
想要在收到弹幕的时候添加一下过滤屏蔽之类的逻辑，我在每一个找到的on_message和on_danmaku方法里都测试了下，但是都没执行。这个到底要加在哪里才行呢？

**Comment by xfgryujk** [comment, marked as answer]
https://github.com/xfgryujk/blivechat/blob/06bd43a173c0d6a35be8c344ae852cce7c4b8b17/frontend/src/views/Room.vue#L379

**Reply by nudoo** [reply]
ok，搞定了，感谢大佬
